### ISSUE TITLE
Issues with 0.2.0

### ISSUE THREAD

**Issue by podviaznikov** [funcool/cuerdas#6]
Hi,

After upgrading to `0.2.0` I see some serious issues.

E.x.
```
(require '[cuerdas.core :as str])

(str/contains? "hello" "anton")
=>true
```

**Comment by dwwoelfel** [comment]
Looks like the tests wouldn't have caught it, but it might be nice to add [automated testing](https://circleci.com/gh/dwwoelfel/cuerdas/8). Happy to make a PR if you like.

**Comment by niwibe** [comment]
I will be happy receive that pr! And sorry for this awful bug!

**Comment by podviaznikov** [comment]
@niwibe no problems at all. Thanks for fixing it so fast.

**Comment by niwibe** [comment]
@podviaznikov Now released 0.3.0 with minor improvements and some bugfixes, including this one.
